### PR TITLE
Improving description

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/6953

### CODE REVIEW

**Review by dariomws — APPROVED** [review]
I approve it but not sure about the note. You guys are the SP experts.

SharePoint Online Preview?

14 and 15 are the only possible values?

**Comment by get-itips** [comment]
> I approve it but not sure about the note. You guys are the SP experts.
> 
> SharePoint Online Preview?
> 
> 14 and 15 are the only possible values?

SharePoint Online Preview must be a very old naming... what do you think @tseward ?

**Comment by tseward** [comment]
You can't create v14 (or v14.5) sites anymore. Tbh this is a parameter that should probably be removed.  But don't use the word "preview". In fact, saying v15 == 2013 is also inaccurate. v15 is 2013 and higher, but a version here has no meaning in SPO.

@trent-green can you ping the PG on potentially removing this parameter in an upcoming release?

File: sharepoint/sharepoint-ps/sharepoint-online/New-SPOSite.md [hunk 1]
```diff
@@ -134,7 +133,7 @@ Accept wildcard characters: False
 
 ### -ResourceQuota
 
-Specifies the quota for this site collection in Sandboxed Solutions units. This value must not exceed the company's aggregate available Sandboxed Solutions quota. The default value is 0. For more information, see Resource Usage Limits on Sandboxed Solutions in SharePoint (<https://msdn.microsoft.com/en-us/library/gg615462.aspx).> Note that this parameter is now obsolete and has been deprecated.
+Specifies the quota for this site collection in Sandboxed Solutions units. This value must not exceed the company's aggregate available Sandboxed Solutions quota. The default value is 0. For more information, see [Resource Usage Limits on Sandboxed Solutions in SharePoint](https://msdn.microsoft.com/library/gg615462.aspx). Note that this parameter is now obsolete and has been deprecated.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/New-SPOSite.md:136]: Change the link to:

https://docs.microsoft.com/previous-versions/office/developer/sharepoint-2010/gg615462(v=office.14)

**Review by tseward — CHANGES_REQUESTED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by tseward** [comment]
@get-itips coming back to this, I'm wondering if a Note: of "only a value of v15 is supported" is appropriate, or "will work". It might be worth some testing.

I'm just not a fan of the description of the cmdlet as it seems to indicate that SPO supports these values in some meaningful way, i.e. that you can create v14.5 sites.

File: sharepoint/sharepoint-ps/sharepoint-online/New-SPOSite.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Example 3 creates a new site collection for the current company with specified s
 
 Specifies the version of templates to use when you are creating a new site collection. This value sets the initial CompatibilityLevel value for the site collection. When this parameter is not specified, the CompatibilityLevel will be by default the highest possible version for the Web application.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/New-SPOSite.md:66]: I would replace this explainer text with something akin to:

"This parameter no longer has any effect and only accepts a value of '15'".

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**